test/integration/scheduled-action-integration.ts

```diff
@@ -0,0 +1,346 @@
+      expect(updatedAction.action).to.eql('unpublish')
+      expect(updatedAction.scheduledFor).to.deep.equal({
+        datetime: updatedSchedule,
+        timezone: 'Europe/Kiev',
+      })
+
+      // cleanup
+      await updatedAction.delete()
+    })
+
+    test('fails to update unsupported property of scheduled actions', async () => {
+      const updatedSchedule = new Date(new Date(datetime).getTime() + ONE_DAY_MS).toISOString()
+
+      const scheduledAction = await testSpace.createScheduledAction({
+        entity: makeLink('Asset', asset.sys.id),
+        environment: makeLink('Environment', environment.sys.id),
+        action: 'unpublish',
+        scheduledFor: {
+          datetime,
+        },
+      })
+
+      expect(scheduledAction.entity).to.eql(makeLink('Asset', asset.sys.id))
+      expect(scheduledAction.scheduledFor).to.deep.equal({
+        datetime,
+      })
+
+      const { sys, ...payload } = scheduledAction
+
+      try {
+        await testSpace.updateScheduledAction({
+          scheduledActionId: sys.id,
+          version: sys.version,
+          payload: {
+            entity: makeLink('Entry', TestDefaults.entry.testEntryId),
+            environment: makeLink('Environment', environment.sys.id),
+            action: 'publish',
+            scheduledFor: {
+              datetime: updatedSchedule,
+            },
+          },
+        })
+      } catch (error) {
+        const parsedError = JSON.parse(error.message)
+        expect(parsedError.status).to.eql(400)
+        expect(parsedError.statusText).to.eql('Bad Request'),
+          expect(parsedError.message).to.eql(
+            'Can only update scheduleFor.datetime and scheduleFor.timezone properties'
+          )
+      }
+
+      // cleanup
+      await scheduledAction.delete()
+    })
+
+    test('delete scheduled action', async () => {
+      const scheduledAction = await testSpace.createScheduledAction({
+        entity: makeLink('Asset', asset.sys.id),
+        environment: makeLink('Environment', environment.sys.id),
+        action: 'unpublish',
+        scheduledFor: {
+          datetime,
+        },
+      })
+
+      // Deletes a Release and all its Release Actions
+      await scheduledAction.delete()
+
+      const response = await testSpace.getScheduledActions({
+        'environment.sys.id': scheduledAction.environment.sys.id,
+        'sys.status': 'scheduled',
+      })
+      expect(response.items.length).to.equal(0)
+    })
+  })
+
+  describe('PlainClient', () => {
+    const defaultParams = {
+      environmentId: TestDefaults.environmentId,
+      spaceId: TestDefaults.spaceId,
+    }
+
+    test('lifecycle', async () => {
+      const plainClient = getPlainClient(defaultParams)
+      const entry = await plainClient.entry.get({
+        entryId: TestDefaults.entry.testEntryReferenceId,
+      })
+
+      const action = await plainClient.scheduledActions.create(
+        {},
```

Comment:
what's the `{}` for? 🤔

Reply:
for defaults like space id and env id. I guess the fallbacks wasn't provided so it always requires something (even an empty object 😬 ). I can change that but would be considered a breaking change? Maybe not?

Reply:
But if this is breaking, we can leave for another PR

Reply:
I checked and it's expected to work this way. It's the same for releases and other entities as well. There people just populated it with default values while in my case default values are internally taken from the sdk client